Sources/Action.swift

```diff
@@ -37,13 +44,17 @@ public final class Action<Input, Output, Error: Swift.Error> {
 	///
 	/// In other words, this sends every value from every unit of work that the `Action`
 	/// executes.
-	public let values: Signal<Output, NoError>
+	public private(set) lazy var values: Signal<Output, NoError> = { [unowned self] in
```

Comment:
Can we make this `lazy` change in a separate PR? I'm not sold on it, and it's not essential to this PR.